site/gatsby-site/src/components/incidents/IncidentForm.js

```diff
@@ -60,6 +110,58 @@ function IncidentForm() {
         <Form.Label>Editors</Form.Label>
         <TagsControl name="editors" />
       </Form.Group>
+
+      <div id="similar-incidents">
+        <RelatedIncidentsArea
+          columnKey={'editor_similar_incidents'}
+          header={'Manually-selected similar and dissimilar incidents'}
+          reports={editorSimilarIncidentReports}
+          loading={false}
+          editable={true}
+          editId={false}
+          error={false}
+        />
+
+        <SemanticallyRelatedIncidents incident={values} editable={true} editId={false} />
+
+        <Form.Group className="mt-3">
+          <Form.Label>Similar Incident Id</Form.Label>
+          <Form.Control
+            type="number"
+            data-cy="similar-id-input"
+            onChange={async (event) => {
```

Comment:
This looks like it would benefit from debouncing. I mean,  otherwise will fire on every keystroke?

Reply:
Got it.